nifi-nar-bundles/nifi-evtx-bundle/nifi-evtx-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,35 @@
+nifi-evtx-nar
+Copyright 2016 The Apache Software Foundation
+
+This product includes the following work from python-evtx (https://github.com/williballenthin/python-evtx):
```

Comment:
i take that back.  They are needed.  But would rephrase to

This contains includes derived works from the Apache Software License V2 library python-evtx (https://github.com/williballenthin/python-evtx) 
Copyright 2012, 2013 Willi Ballenthin william.ballenthin@mandiant.com
while at Mandiant http://www.mandiant.com
The derived work is adapted from Evtx/Evtx.py, Evtx/BinaryParser.py,Evtx/Nodes.py and can be found in the org.apache.nifi.processors.evtx.parser package.